### PR TITLE
connecting db up to front end w/ seeded db

### PR DESCRIPTION
**Pull request by CooperDoten**
connecting up that eventService so we can display events from DB

### CODE REVIEW

File: packages/client/src/services/eventsService.ts [hunk 1]
```diff
@@ -57,7 +59,7 @@ export default class EventsService {
     docsRefs.docs.forEach((doc) => {
       const event = doc.data() as Event;
       const { categories } = event;
-
+      console.log("event", event);
```
Comment (adamschnaare) [inline packages/client/src/services/eventsService.ts:62]: Remove the console.logs please

File: packages/client/src/services/usersService.ts [hunk 2]
```diff
@@ -44,9 +44,9 @@ export default class UsersService {
   public async getUser(uid: string) {
     const doc = await this.db.collection(this.collection).doc(uid).get();
     const data = doc.data() as User;
-    const user = UserFactory(data);
+    console.log(data)
```
Comment (adamschnaare) [inline packages/client/src/services/usersService.ts:47]: Remove the console.logs please

File: packages/client/src/views/CreateEvent.tsx [hunk 3]
```diff
@@ -171,7 +171,7 @@ export default function CreateEventView() {
       ...state,
       categories,
     };
-
+    console.log(event);
```
Comment (adamschnaare) [inline packages/client/src/views/CreateEvent.tsx:174]: Remove the console.logs please

File: packages/client/src/views/EventDetails.tsx [hunk 4]
```diff
@@ -98,7 +98,10 @@ export default function EventDetailsView() {
   }
 
   useEffect(() => {
-    eventService.getEvent(getEventIdFromURL()).then((event) => {
+    const getEvent = getEventIdFromURL();
```
Comment (adamschnaare) [inline packages/client/src/views/EventDetails.tsx:101]: Remove the console.logs please

File: packages/client/src/views/Events.tsx [hunk 5]
```diff
@@ -101,17 +105,19 @@ const useStyles = makeStyles((theme: Theme) =>
...
     return str.length > n ? str.substr(0, n - 1) + "..." : str;
   };
 
-  // const onClick = (event?) => {
-  //   setSelectedEventState(event);
-  //   history.push(routes.EVENT_DETAILS_URL + selectedEventState.id);
-  // };
+  const onClick = (event?) => {
+    setSelectedEventState(event);
+    console.log(event);
```
Comment (adamschnaare) [inline packages/client/src/views/Events.tsx:118]: Remove the console.logs please

File: packages/client/utils/seed.mjs [hunk 6]
```diff
@@ -22,7 +21,7 @@ const generateEvents = (numberOfEvents) => {
 };
 //this function used to help randomly generate category data
 const randomCategories = () => {
-  const fullList = ["marketing", "leadership", "finance", "technology"];
+  const fullList = ["marketing", "leadership", "finance", "it"];
```
Comment (adamschnaare) [inline packages/client/utils/seed.mjs:24]: A note for future - would love to see these represented in either a `model` or a `constant` to be referenced everywhere.

File: packages/client/src/models/event.ts [hunk 7]
```diff
@@ -45,9 +45,9 @@ export function formatCentsToCurrency(
 }
 
 export function displayEventDate(date) {
-    return new Date(date).toDateString();
-  }
+  return new Date(date).toDateString();
```
Comment (adamschnaare) [inline packages/client/src/models/event.ts:48]: Let's have a discussion on what format we want to use. A popular library that I usually go to is `moment.js` https://momentjs.com/

**Review by adamschnaare — CHANGES_REQUESTED** [review]

File: packages/client/src/services/eventsService.ts (hunk 1 above)
Comment (CooperDoten) [reply packages/client/src/services/eventsService.ts]: done and done

File: packages/client/src/services/usersService.ts (hunk 2 above)
Comment (CooperDoten) [reply packages/client/src/services/usersService.ts]: done and done

File: packages/client/src/views/CreateEvent.tsx (hunk 3 above)
Comment (CooperDoten) [reply packages/client/src/views/CreateEvent.tsx]: donezo

File: packages/client/src/views/Events.tsx (hunk 5 above)
Comment (CooperDoten) [reply packages/client/src/views/Events.tsx]: yaaaaa

File: packages/client/utils/seed.mjs (hunk 6 above)
Comment (CooperDoten) [reply packages/client/utils/seed.mjs:24]: 100% should be a model to draw from

**Review by CooperDoten — COMMENTED** [review]
I think I have submitted all the necessary changes -- lmk also, didn't know if this was something usually resolved up here in github or if its more normal to do it on a branch and repush?

**Review by adamschnaare — APPROVED** [review]

**Pull request merged**